styles/banner.css

```diff
@@ -0,0 +1,40 @@
+.Cart {
+  font-family: sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+p:not(:last-of-type) {
+  margin-bottom: 0.5rem;
+}
+
+.banner {
+  display: flex;
+  padding: 1rem;
+}
+
+.banner_container {
+  width: auto;
+  font-family: sans-serif;
```

Comment:
Same, please don't override the font-family use the Typography component instead :)